### DISCUSSION TITLE
github auth?

### DISCUSSION THREAD

**Discussion by DanielFGray** [CodinGame/monaco-vscode-api#547] [Q&A]
vscode typically ships with github auth, is there a way to hook into it here? ive struggled finding examples of using it in "normal" vscode

**Comment by CGNonofr** [comment]
I've never tried to use it, but I think the github auth comes with one of the default extension. you also probably need a few service overrides (at least the authentication one)

**Comment by TheUltDev** [comment]
@DanielFGray did you make any breakthrough with this? about to start the journey myself.

**Reply by DanielFGray** [reply]
no, I've poked around and prodded, but never got it working. also struggled migrating to more recent versions, which has made further poking and prodding more difficult
